docs/cudf/source/api_docs/groupby.rst

```diff
@@ -48,15 +48,24 @@ Computations / descriptive stats
    GroupBy.cummax
    GroupBy.cummin
    GroupBy.cumsum
+   GroupBy.diff
    GroupBy.ffill
+   GroupBy.first
+   GroupBy.get_groups
```

Comment:
```suggestion
```

Grep shows we don't have a `get_groups` yet on the python side. But we have a c++ API for it.

Reply:
never mind you probably want `get_group` here